### DISCUSSION TITLE
Code review IP92 Alexey Kharenko

### DISCUSSION THREAD

**Discussion by AlexKharenko** [HowProgrammingWorks/LiveQA#38] [Code review]
https://github.com/AlexKharenko/ChatApp/pull/1

**Comment by tshemsedinov** [comment]
See first review: https://github.com/AlexKharenko/ChatApp/pull/1

**Comment by tshemsedinov** [comment]
We have video recording of 1st review, see in telegram of your group

**Comment by AlexKharenko** [comment]
https://github.com/AlexKharenko/ChatApp/pull/2

**Comment by AlexKharenko** [comment]
https://github.com/AlexKharenko/ChatApp/pull/3
Final version of backend for chatapp
